common/txmgr/types/forwarder_manager.go

```diff
@@ -0,0 +1,12 @@
+package types
+
+import (
+	"github.com/smartcontractkit/chainlink/core/services"
+)
+
+type ForwarderManager[ADDR any] interface {
```

Comment:
nit/ Does the `*Manager` suffix add information?
```suggestion
type Forwarder[ADDR any] interface {
```

Reply:
It's not a forwarder by itself but it manages getting forwarders